.github/workflows/tests.yml

```diff
@@ -15,6 +15,10 @@ jobs:
         php: [ 7.2, 7.3, 7.4, '8.0', 8.1 ]
         laravel: [ 6, 7, 8, 9 ]
         stability: ['prefer-lowest', 'prefer-stable']
+        include:
+          - php: 8.2
+            laravel: 9
+            stability: prefer-stable --ignore-platform-req=php+
```

Comment:
This line isn't needed anymore. PHP 8.2 is stable enough to start requiring other packages to release compatible versions. We've also removed this from the Laravel framework build and have a passing build. In fact, 8.2 can be added to the list of PHP versions a bit above here. It needs to be excluded for Laravel 6, 7 & 8 in the exclude list below.

Reply:
Just this line (line 21)? Or more?